packages/format-library/src/language/index.js

```diff
@@ -0,0 +1,115 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * WordPress dependencies
+ */
+import { RichTextToolbarButton } from '@wordpress/block-editor';
+import {
+	TextControl,
+	SelectControl,
+	Button,
+	Popover,
+} from '@wordpress/components';
+import { useState } from '@wordpress/element';
+import { applyFormat, removeFormat, useAnchor } from '@wordpress/rich-text';
+import { translation } from '@wordpress/icons';
+
+const name = 'core/language';
+const title = __( 'Language' );
+
+const Edit = ( props ) => {
+	const { contentRef, isActive, onChange, value } = props;
+	const anchorRef = useAnchor( {
+		editableContentElement: contentRef.current,
+		language,
+	} );
+
+	const [ lang, setLang ] = useState( '' );
+	const [ dir, setDir ] = useState( 'ltr' );
+
+	const [ isPopoverVisible, setIsPopoverVisible ] = useState( false );
+	const togglePopover = () => {
+		setIsPopoverVisible( ( state ) => ! state );
+		setLang( '' );
+		setDir( 'ltr' );
+	};
+
+	return (
+		<>
+			<RichTextToolbarButton
+				icon={ translation }
+				label={ title }
+				title={ title }
+				onClick={ () => {
+					if ( isActive ) {
+						onChange( removeFormat( value, name ) );
+					} else {
+						togglePopover();
+					}
+				} }
+				isActive={ isActive }
+			/>
+
+			{ isPopoverVisible && (
+				<Popover
+					className="components-lang-attribute-popover"
+					anchor={ anchorRef }
+					placement="bottom"
+				>
+					<TextControl
+						label={ title }
+						value={ lang }
+						onChange={ ( val ) => setLang( val ) }
+						aria-describedby={ `language-selector-describe-${ contentRef.current.id }` }
```

Comment:
This should be removed as this component generates `aria-describedby`. Currently, the ID is invalid.

Reply:
You're absolutely right, I forgot to remove that when I changed the implementation 👍 
Done now.